models/error.go

```diff
@@ -211,6 +211,21 @@ func (err ErrReachLimitOfRepo) Error() string {
 	return fmt.Sprintf("user has reached maximum limit of repositories [limit: %d]", err.Limit)
 }
 
+// ErrReachLimitOfPrivateRepo represents a "ReachLimitOfRepo" kind of error.
```

Comment:
s/ReachLimitOfRepo/ReachLimitOfPrivateRepo/

Though I don't really understand the comment this is copied from, there is no `ReachLimitOfRepo` struct or otherwise.